simulator/src/main/java/com/hedera/block/simulator/mode/PublisherModeHandler.java

```diff
@@ -120,7 +123,11 @@ private void constantRateStreaming() throws InterruptedException, IOException, B
                 break;
             }
 
-            publishStreamGrpcClient.streamBlock(block);
+            final boolean streamSuccess = publishStreamGrpcClient.streamBlock(block);
+            if (!streamSuccess) {
+                LOGGER.log(System.Logger.Level.INFO, "Block Stream Simulator stopped streaming due to errors.");
+                break;
+            }
```

Comment:
I think you can skip the boolean variable assignment (see comment above)

Reply:
Changed, although I usually like to assign, just for better readability.